index.html

```diff
@@ -1,10 +1,138 @@
 <!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="UTF-8">
+<html lang="en" xmlns="http://www.w3.org/1999/html">
+  <h1>Моя первая страница</h1>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="description" content="IT school from Belarus" />
+    <meta name="keywords" content="HTML,CSS,Javascript,React" />
+    <meta name="author" content="Irina Breiksh" />
+    <meta name="viewport" content="width=device-width,initial-scale=1.0" />
     <title>Title</title>
-</head>
-<body>
-
-</body>
-</html>
\ No newline at end of file
+  </head>
+  <body>
+    <hr />
+    <img src="./images/matrix.jpg" alt="coding matrix" />
+    <ol>
+      <li>Знакомство с Web
+        <ul>
+          <li>Краткая история Web и как устроен интернет</li>
+          <li>Как устроен браузер и как он получает страницы по сети</li>
+          <li>Что такое Git</li>
+          <li>Создание первого репозитория</li>
+          <li>О файле .gitignore</li>
+        </ul>
+      </li>
+      <li>HTML-вёрстка – первые шаги
+        <ul>
+        <li>Настройка рабочей среды</li>
+        <li>Краткая история HTML</li>
+        <li>Типы тегов и атрибуты</li>
+        <li>Теги для работы с текстом</li>
+        <li>Создание собственной страницы</li>
+        <li>Основы ветвления в Git</li>
+        <li>Создание pull request на GitHub</li>
+      </ul>
```

Comment:
Close tag should be on the same level as open tag.

Reply:
Ok, changed